Generals/Code/GameEngine/Source/GameClient/GUI/GUICallbacks/Menus/OptionsMenu.cpp

```diff
@@ -1229,7 +1229,7 @@ static void saveOptions( void )
 	//-------------------------------------------------------------------------------------------------
 	// Set System Time Font Size
 	val = TheWritableGlobalData->m_systemTimeFontSize; // TheSuperHackers @todo replace with options input when applicable
```

Comment:
I think this is still wrong because the value is not loading from and saving to the OptionPreferences during game runtime. To test this, edit the Options.ini while the game runs and press ACCEPT button in the Options Menu.

Check how I setup "MoneyTickVolume" in #1353. I think it is correct there. (I tested it.)